### PR TITLE
add the text "The first publicly available description of HTML was a …

### PR DESCRIPTION
**Pull request by karlwil1**
…document called "HTML Tags", first mentioned on the Internet by Tim Berners-Lee in late 1991. Who is widely credited with the invention of html." to article

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by karlwil1** [comment]
> "The first publicly available description of HTML was a document called "HTML Tags", first mentioned on the Internet by Tim Berners-Lee in late 1991" - from https://en.wikipedia.org/wiki/HTML

File: guide/english/html/index.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ title: HTML
 HyperText Markup Language (HTML) is a markup language used to construct online documents and is the foundation of most websites today. A markup language like HTML allows us to 1) create links to other documents, 2) structure the content in our document, and 3) ascribe context and meaning to the content of our document.
  
 An HTML document has two aspects to it. It contains structured information (Markup), and text-links (HyperText) to other documents. We structure our pages using [HTML elements](#). They are constructs of the language providing [structure](#) and [meaning](#) in our document for the browser and the [<anchor>](#) element links to other documents across the internet.
- 
+
+The first publicly available description of HTML was a document called "HTML Tags", first mentioned on the Internet by Tim Berners-Lee in late 1991. Who is widely credited with the invention of html.
```
Comment (Manish-Giri) [inline guide/english/html/index.md]:
```suggestion
The first publicly available description of HTML was a document called "HTML Tags", first mentioned on the Internet in late 1991 by Tim Berners-Lee, who is widely credited with the invention of html. 
```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
Suggested changes to be addressed.

**Review by Manish-Giri — APPROVED** [review]
Hi @karlwil1 ,

 Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍 We're happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**